index.html

```diff
@@ -14,15 +14,26 @@
     />
   </head>
   <body>
+    
     <header class="navbar">
       <a href="#" class="logo">My Logo</a>
-      <nav><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
+      <nav id = "menu-icon"><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
       <ul class="menu-links">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
+        <li><a href="#projects-section">Portfolio</a></li>
+        <li><a href="#about-intro">About</a></li>
+        <li><a href="#contact-me">Contact</a></li>
       </ul>
```

Comment:
- [ ] Well done implementing this part of the menu links but let's not forget that mylogo, the menu icon as well as the nav list all  should be wrapped inside a `nav` tag,  because that's what makes the navigation in the first place 👍

Reply:
done